tema14Ianuarie/src/com/sci/bv/tema14Ianuarie/AnimalProduct.java

```diff
@@ -0,0 +1,102 @@
+package com.sci.bv.tema14Ianuarie;
+
+
+/**
+ * In this Class are stocked animal products and their characteristics.
+ * (ID, quantity,name of product, list of vitamins, price, validity, weight)
+ */
+
+public class AnimalProduct {
```

Comment:
A lot of the attributes are the same between AnimalProduct and VegetableProduct. Why not have a parent class Product with the shared attributes an behavior?